client/my-sites/customer-home/main.jsx

```diff
@@ -87,6 +92,21 @@ const Home = ( {
 	const siteDomains = useSelector( ( state ) => getDomainsBySiteId( state, siteId ) );
 	const customDomains = siteDomains?.filter( ( domain ) => ! domain.isWPCOMDomain );
 	const customDomain = customDomains?.length ? customDomains[ 0 ] : undefined;
+	const primaryDomain = customDomains?.length
+		? customDomains.find( ( domain ) => domain.isPrimary )
+		: undefined;
```

Comment:
```suggestion
	const primaryDomain = customDomains?.find( ( domain ) => domain.isPrimary );
```
I think this should work?